src/polkadot/augment-api-tx.ts

```diff
@@ -212,7 +212,7 @@ declare module '@polkadot/api/types/submittable' {
         [Ticker, Balance, PortfolioId]
       >;
       /**
-       * Initializes a new token, with the initiating account as its owner.
+       * Initializes a new security, with the initiating account as its owner.
```

Comment:
```suggestion
       * Initializes a new security token, with the initiating account as its owner.
```
That's the original comment